aws/sdk/examples/dynamodb/src/bin/list-items.rs

```diff
@@ -58,11 +59,18 @@ async fn main() -> Result<(), Error> {
         println!();
     }
 
-    let resp = client.scan().table_name(table).send().await?;
+    let items: Result<Vec<_>, _> = client
+        .scan()
+        .table_name(table)
+        .paginate()
+        .items()
```

Comment:
In cases where a user just wants all the data and doesn't care about pages, do they still need to paginate?

Reply:
I don't follow, if you want all the data and don't care about pages, you could use `collect()` (which will drive the paginator to completion but you won't have to interact with the pages)

Reply:
What I meant is that I see two use cases:
- I want to look through the data page by page and potentially load less total data
- I just want it all

In the second case, do they need to create the paginated operation at all or can they just send the operation they have?

Reply:
if you send the operation you have, you only get the first page

Reply:
Ah! I understand now